base/src/account/mod.rs

```diff
@@ -52,3 +55,31 @@ pub struct Account {
     /// TODO: This should contain a Snapp account data once we have something to test against
     pub snapp: Option<()>,
 }
+
+impl mina_hasher::Hashable for Account {
+    type D = ();
+
+    // Uncomment these fields once they have implemented Hashable trait
+    // and add unit tests when it's complete
+    fn to_roinput(&self) -> ROInput {
+        let mut roi = ROInput::new();
+        roi
+            // .append_hashable(self.public_key)
+            .append_hashable(&self.token_id)
+            // .append_hashable(self.token_permissions)
+            .append_hashable(&self.balance)
+            // .append_hashable(self.nonce)
+            // .append_hashable(self.receipt_chain_hash)
+            // .append_hashable(self.delegate)
+            // .append_hashable(self.voting_for)
+            // .append_hashable(self.timing)
+            // .append_hashable(self.permissions)
+            // .append_hashable(self.snapp)
```

Comment:
Once #197 is merged I think there are no further blockers for this right?

Reply:
Yeah I believe so